packages/core/src/App/Containers/AccountSwitcher/account-switcher.jsx

```diff
@@ -138,19 +138,19 @@ class AccountSwitcher extends React.Component {
             {
                 account_types: ['svg'],
                 icon: 'Synthetic',
-                title: localize('Synthetic'),
+                title: localize('DMT5 Synthetic'),
                 type: 'synthetic',
             },
             {
                 account_types: ['vanuatu', 'svg_standard'],
                 icon: 'Financial',
-                title: localize('Financial'),
+                title: localize('DMT5 Financial'),
                 type: 'financial',
             },
             {
                 account_types: ['labuan'],
                 icon: 'Financial STP',
-                title: localize('Financial STP'),
+                title: localize('DMT5 Financial STP'),
```

Comment:
why are we adding these prefixes again? the title says DMT5 accounts, so the accounts were updated to remove DMT5 repetition

Reply:
It was in the Design, What you are saying makes more sense, I'll remove it.